### ISSUE TITLE
Ability to add custom script phases to user project

### ISSUE THREAD

**Issue by anivaros** [CocoaPods/CocoaPods#9565]
I have third-party static framework, that should be integrated with custom script in user project.
I want to distribute that framework in my internal projects via cocoa pods.
It would be nice to have ability to integrate custom scripts not only in pods projects, but in user project too.

I've found some related code in [cocoapods/installer/user_project_integrator/target_integrator.rb](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/installer/user_project_integrator/target_integrator.rb) `create_or_update_user_script_phases`, but it looks like I can't use it via podspec.

**Comment by dnkoutso** [comment]
Yeap could be an enhancement although it is a bit scary to give the ability for someone to do that.

**Comment by anivaros** [comment]
But what difference between custom scripts in pods project which already implemented and in user project?

**Comment by dnkoutso** [comment]
yeah I agree with you not much difference. We warn today that a pod installed a bunch of script phases.

**Comment by samdmarshall** [comment]
this has been discussed at length before, and i believe we settled on not allowing this as it would only cause severe complications to the cocoapods users and give individual pod owners the power to break or push malicious code directly into other people's project files without warning -- not to mention breaking the user's project targets by messing with build phase ordering and adding a bunch of new build phases.

**Comment by anivaros** [comment]
But you could make it work only user explicitly enable this in podfile

**Comment by dnkoutso** [comment]
That could work with `s.user_script_phase` or something.
